docs/pages/what-is-fable-server.md

```diff
@@ -0,0 +1,9 @@
+# Fable Server
```

Comment:
In another issue, @Pauan used `daemon` instead of `server`. I was thinking that maybe this is a more appropriate term as some users had some confusion and thought the Fable server was a local static server similar to Webpack's.